cmd/subctl/deploybroker.go

```diff
@@ -70,7 +70,8 @@ func addDeployBrokerFlags() {
 	deployBroker.PersistentFlags().StringVar(&deployflags.Repository, "repository", "", "image repository")
 	deployBroker.PersistentFlags().StringVar(&deployflags.ImageVersion, "version", "", "image version")
 
-	deployBroker.PersistentFlags().BoolVar(&deployflags.OperatorDebug, "operator-debug", false, "enable operator debugging (verbose logging)")
+	deployBroker.PersistentFlags().BoolVar(&deployflags.OperatorDebug, "operator-debug", false,
+		"enable operator debugging (verbose logging)")
```

Comment:
Why was this changed?

Reply:
Leftovers from the support for both pretty-logging and klog logging.  Reverted.